packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -372,6 +379,34 @@ export default function PostOptionsMenu({
     }
   }
 
+  if (isLoggedIn && post?.author) {
+    const authorName = post.author.name || post.author.username;
+    const isFollowingUser = [
+      ContentPreferenceStatus.Follow,
+      ContentPreferenceStatus.Subscribed,
+    ].includes(post.author.contentPreference?.status);
+
+    postOptions.push({
+      icon: <MenuIcon Icon={AddUserIcon} />,
+      label: `${isFollowingUser ? 'Unfollow' : 'Follow'} ${authorName}`,
+      action: () => {
+        if (!isFollowingUser) {
+          follow({
+            id: post.author.id,
+            entity: ContentPreferenceType.User,
+            entityName: authorName,
+          });
+        } else {
+          unfollow({
+            id: post.author.id,
+            entity: ContentPreferenceType.User,
+            entityName: authorName,
```

Comment:
Also I did `@name` (probably should remove the @) but maybe we can have some rules around it internally?

Reply:
yep dc3b29712bdfd86cf5408cfeb6bcada7606e0630